src/components/Trends/TrendsPanel.jsx

```diff
@@ -174,9 +174,9 @@ export class TrendsPanel extends React.Component {
           <div className="layout-row">
             <section className="chart">
               <h2 className="area-chart-title">{this._areaChartTitle()}</h2>
-              <p className="chart-helper-text">A time series graph of
-               complaint counts for the selected date range.
-                Hover on the chart to see the count for each date interval.
+              <p className="chart-helper-text">A time series graph of the
+              (up to) five highest volume complaint counts for the selected
```

Comment:
This doesn't sound right.

Reply:
never mind. that is correct when i see it all on one line.  it looks a bit confusing with a line break.